appcircle_ios_reactnative_cache_push/1.0.2/component.yaml

```diff
@@ -4,7 +4,7 @@ displayName: "Cache Push"
 description: "Uploads user selected files and folders to Appcircle cache."
 docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps#cache-push
 requiredComponents: 
```

Comment:
Unfortunately `Cache Push` step is depended with `Git Clone`.
You can check this dependency => https://github.com/appcircleio/appcircle-cache-push-component/blob/ab0c2ac8510968b3b50af8a917075e409853fe0c/main.rb#L210
Can you made this like before:
```
requiredComponents: "appcircle_git_clone"
```